apps/examples/Form/KeyboardAware/index.css

```
@@ -20,9 +20,9 @@
   width: 100%;
   height: 100%;
 
-  padding: 36px;
-  padding-left: 16px;
-  padding-right: 16px;
+  padding: 64px;
```

Stylelint (17.4.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)